### PR TITLE
Bitcoin: add configuration to broadcast through a custom RPC node

### PR DESCRIPTION
**Pull request by KhalilBellakrid**
So this is more and more asked by community to use their own RPC nodes (or ones they trust)

### CODE REVIEW

File: core/idl/wallet/configuration.djinni [hunk 1]
```diff
@@ -66,6 +66,8 @@ Configuration = interface +c {
     const BLOCKCHAIN_EXPLORER_ENGINE: string = "BLOCKCHAIN_EXPLORER_ENGINE";
     # Sets the API endpoint for API based engine (Ledger's API).
     const BLOCKCHAIN_EXPLORER_API_ENDPOINT: string = "BLOCKCHAIN_EXPLORER_API_ENDPOINT";
+    # Sets the node's endpoint to broadcast transactions with one's node
+    const BLOCKCHAIN_NODE_ENDPOINT: string = "BLOCKCHAIN_NODE_ENDPOINT";
```
Comment (phaazon) [inline core/idl/wallet/configuration.djinni]: Seems fine to me, but is this used only for Bitcoin? Does it make sense to use the overall configuration for that?

File: core/src/wallet/bitcoin/explorers/BitcoinLikeBlockchainExplorer.cpp [hunk 2]
```diff
@@ -36,5 +36,22 @@ namespace ledger {
...
+                                                                            const std::string &rpcNode) {
+            std::stringstream body;
+            body << '{"jsonrpc": "2.0", "id": "broadcastTx", "method": "sendrawtransaction", "params": ["' << hex::toString(rawTx) <<'"] }';
+            auto bodyString = body.str();
+            return client->POST("",
+                               std::vector<uint8_t>(bodyString.begin(), bodyString.end()),
+                                                    std::unordered_map<std::string, std::string>{},
+                                                    rpcNode)
+                    .json().template map<String>(context, [] (const HttpRequest::JsonResult& result) -> String {
+                        auto& json = *std::get<1>(result);
```
Comment (alekece) [inline core/src/wallet/bitcoin/explorers/BitcoinLikeBlockchainExplorer.cpp]: This line seems SU-PER dangerous to me. 

Can you left a comment describing the result json scheme ? That will ease the debugging if an error occurred in this portion of code.

Comment (phaazon) [reply core/src/wallet/bitcoin/explorers/BitcoinLikeBlockchainExplorer.cpp]: Agreed. What happens when the parsing fails?

**Comment by phaazon** [comment]
`HODL` for stalling PR.

**Comment by pollastri-pierre** [comment]
Closing this one, no demand right now.

**Pull request closed without merging**